package.json

```diff
@@ -137,6 +143,7 @@
     ]
   },
   "resolutions": {
-    "babel-loader": "8.1.0"
+    "babel-loader": "8.1.0",
+    "bn.js": "4.12.0"
```

Comment:
Required to resolve polkadot-types type generation issues, see https://github.com/polkadot-js/api/issues/4024